src/plugins/workspace/server/plugin.ts

```diff
@@ -83,8 +88,35 @@ export class WorkspacePlugin implements Plugin<{}, {}> {
     if (isPermissionControlEnabled) {
       this.permissionControl = new SavedObjectsPermissionControl(this.logger);
 
+      this.logger.info('Dynamic application configuration enabled:' + !!applicationConfig);
+      if (!!applicationConfig) {
+        core.http.registerOnPostAuth(async (request, response, toolkit) => {
+          const [coreStart] = await core.getStartServices();
```

Comment:
If there is no auth info post auth, we should fail fast and do not call the `getEntityConfig` api.

Reply:
Yes, I have modified the logic.